nixos/modules/services/desktops/gnome/gnome-keyring.nix

```diff
@@ -1,53 +1,53 @@
 # GNOME Keyring daemon.
 
-{ config, pkgs, lib, ... }:
-
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.gnome.gnome-keyring;
+in
 {
 
-  meta = {
-    maintainers = lib.teams.gnome.members;
-  };
-
-  ###### interface
-
-  options = {
-
-    services.gnome.gnome-keyring = {
+  meta.maintainers = lib.teams.gnome.members;
 
-      enable = lib.mkOption {
-        type = lib.types.bool;
-        default = false;
-        description = ''
-          Whether to enable GNOME Keyring daemon, a service designed to
-          take care of the user's security credentials,
-          such as user names and passwords.
-        '';
-      };
+  ## Interface ##
```

Comment:
We are removing these comments in GNOME modules when refactoring, since they are redundant with `options` and `config` attributes. We still use `options = {}` and `meta` unpacked to make the sections more visible.